astropy/table/notebook_backends.py

```diff
@@ -0,0 +1,116 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Backend implementations for `~astropy.table.Table` interface
+with IPython notebooks.
```

Comment:
```suggestion
with Jupyter notebooks.
```

Reply:
Fun fact: I started out with "Jupyter" but then changed it back to "IPython" to match existing text... 😅